static/app/actionCreators/tags.tsx

```diff
@@ -192,3 +208,214 @@ export function fetchSpanFieldValues({
     query,
   });
 }
+
+type FetchOrganizationTagsParams = {
+  orgSlug: string;
+  // TODO: Change this to Dataset type once IssuePlatform is added
+  dataset?: string;
+  projectIds?: string[];
+  useCache?: boolean;
+};
+
+export const makeFetchOrganizationTags = ({
+  orgSlug,
+  dataset,
+  projectIds,
+  useCache = true,
+}: FetchOrganizationTagsParams): ApiQueryKey => [
+  `/organizations/${orgSlug}/tags/`,
+  {query: {dataset, useCache, project: projectIds}},
+];
+
+export const useFetchOrganizationTags = (
+  params: FetchOrganizationTagsParams,
+  options: Partial<UseApiQueryOptions<Tag[]>>
+) => {
+  return useApiQuery<Tag[]>(makeFetchOrganizationTags(params), {
+    staleTime: Infinity,
+    keepPreviousData: true,
+    ...options,
+  });
+};
+
+type UseFetchIssueTagsParams = {
+  orgSlug: string;
+  projectIds: string[];
+  useCache?: boolean;
+};
+
+export const useFetchIssueOrganizationTags = ({
+  orgSlug,
+  projectIds,
+  useCache = true,
+}: UseFetchIssueTagsParams) => {
+  const eventsTagsQuery = useFetchOrganizationTags(
+    {
+      orgSlug,
+      projectIds,
+      dataset: Dataset.ERRORS,
+      useCache,
+    },
+    {}
+  );
+
+  const issuePlatformTagsQuery = useFetchOrganizationTags(
+    {
+      orgSlug,
+      projectIds,
+      dataset: 'search_issues',
+      useCache,
+    },
+    {}
+  );
+
+  const eventsTags = [
+    ...(eventsTagsQuery.data || []),
+    ...(issuePlatformTagsQuery.data || []),
+  ];
+
+  const eventsTagCollection: TagCollection = eventsTags.reduce<TagCollection>(
+    (acc, tag) => {
+      acc[tag.key] = {...tag, predefined: false, kind: FieldKind.TAG};
+      return acc;
+    },
+    {}
+  );
+
+  return {
+    tags: eventsTagCollection,
+    isLoading: eventsTagsQuery.isLoading || issuePlatformTagsQuery.isLoading,
+    isError: eventsTagsQuery.isError || issuePlatformTagsQuery.isError,
+  };
+};
+
+export function builtInIssuesFields(
```

Comment:
This function defines and populates the builtin tags. It is nearly the same as the [getIssueAttributes](https://github.com/getsentry/sentry/blob/f66b6a9013dc827900f1f6fd087e51cfff06ce43/static/app/stores/tagStore.tsx#L69) function defined in `TagStore`. We cannot reuse that function because it depends on the TagStore's built in state to populate the `has` field values.